src/backend/TrafficCourts/Workflow.Service/Consumers/EmailVerificationReceivedConsumer.cs

```diff
@@ -38,6 +38,7 @@ public async Task Consume(ConsumeContext<EmailVerificationSuccessful> context)
             Dispute? dispute = await _oracleDataApiService.GetDisputeByNoticeOfDisputeGuidAsync(message.NoticeOfDisputeGuid, context.CancellationToken);
             if (dispute is null)
             {
+                // TODO: how do we handle if the dispute has not been created in the database yet?
                 _logger.LogInformation("Dispute not found");
                 return;
             }
```

Comment:
The code that you added to check whether the disputant's email is verified or not is not committed yet I think.

Reply:
I added a hosted service to fetch all the verified emails and resend the message. It will run once at startup for now.